src/tron.jl

```diff
@@ -148,7 +158,7 @@ function tron(::Val{:Newton},
   end
 
   return GenericExecutionStats(status, nlp, solution=x, objective=fx, dual_feas=πx,
-                               iter=iter, elapsed_time=el_time)
+                               primal_feas=0, iter=iter, elapsed_time=el_time)
```

Comment:
`primal_feas` wasn't automatically 0? Also, it needs to be a `zero(eltype(x))`.

Reply:
It was at Inf!

Reply:
Ah, it's `0` if `unconstrained(nlp)`.